apps/ledger-live-mobile/src/hooks/useSetNavigationHeader.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo, useEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+
+import { navigationRef } from "../rootnavigation";
+
+export type Args = {
+  headerShown?: boolean;
+  title?: string;
+  HeaderLeft?: React.ComponentType | null;
+  HeaderRight?: React.ComponentType | null;
+};
```

Comment:
it would be nice to have the `header` option as well

Reply:
done ;)